src/components/ErrorState/ErrorState.tsx

```diff
@@ -61,7 +52,7 @@ export const ErrorState = ({
 }: ErrorStateProps) => {
   return (
     <Wrapper>
```

Comment:
Melhorar composicao.

Reply:
Fixed in this [commit](https://github.com/joaooliveiradev/conduit-frontend/pull/43/commits/3ddd24dab90e99a2fcb1f3a55ce217254f878505)